### PR TITLE
[使用指南] 7.檢視服務使用歷史紀錄

### PR DESCRIPTION
**Pull request by cxheng315**

### CODE REVIEW

File: Account & Project/1. User guides/7. 檢視服務使用歷史紀錄.md [hunk 1]
```diff
@@ -1 +1,101 @@
...
+
+The TWCC portal can display up to 1000 latest data at once, and the old data will no longer be viewable. You can click "**Export Report**" to save the record in real-time.
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_788bd83e639bf54045d4f0c2b0b0729d.png)
+
+
+
+:::warning
+
+{%hackmd @docsharedstyle/tip-zh %}
```
Comment (VigaWei) [inline Account & Project/1. User guides/7. 檢視服務使用歷史紀錄.md:93]: tip-zh > tip-en

File: Account & Project/1. User guides/7. 檢視服務使用歷史紀錄.md [hunk 2]
```diff
@@ -1 +1,101 @@
...
+
+### Step 1. Enter "History" page
+
+After sign in to the TWCC portal, click "**User Name**"> "**History**".
+
+![](https://cos.twcc.ai/SYS-MANUAL/uploads/upload_ee4629c042f2377a9c425eb9a0ad29a6.png)
+
+### Step 2. Select project member (tenant users can omit)
+
+If you are a tenant admin, please select your name or the name of the member whose record you want to view.
```
[the remark this answers is not in the document]
Comment (cxheng315) [reply Account & Project/1. User guides/7. 檢視服務使用歷史紀錄.md:48]: 2. maybe should be a "member to view the records" is better?

**Review by cxheng315 — COMMENTED** [review]
some opinion

File: Account & Project/1. User guides/7. 檢視服務使用歷史紀錄.md [hunk 3]
```diff
@@ -1 +1,101 @@
...
+- Operation status
+- Operation response time
+- Operation start time
+
+:::info
+
+{%hackmd @docsharedstyle/note-zh %}
+
+[1] **`Tenant User`** are only able to inquire their own usage records;**`Tenant Admin`** are able to inquire all members' usage records.
+[2] TWCC APIs are categorized into  5 types of service methods ([<ins>Get to know about RESTful API </ins>](https://en.wikipedia.org/wiki/Representational_state_transfer))：
```
Comment (VigaWei) [inline Account & Project/1. User guides/7. 檢視服務使用歷史紀錄.md:28]: [SUG]

into  5 types of service methods > into 5 types of service methods

**Review by VigaWei — COMMENTED** [review]
2 more comments, thx!

**Pull request merged**